EIPS/eip-x.md

```diff
@@ -0,0 +1,113 @@
+---
+eip: x
+title: CREATE3
+author: Moody Salem
+category: Core
+type: Standards Track
+status: Draft
+created: 2020-12-22
+---
+
+### Specification
+
+This EIP is a fork of [./eip-1014.md](EIP-1014) which adds an account creation opcode that does not involve the init code.
+
+Adds a new opcode (`CREATE3`) at `0xf6`, which takes 4 stack arguments: endowment, memory_start, memory_length, salt. 
+Behaves identically to `CREATE2` (`0xf5`), except using `keccak256( 0xfe ++ address ++ salt )[12:]`,
+which does not include the hash of the init code.
+
+The `CREATE3` has the same `gas` schema as `CREATE2`, but with one fewer `GSHA3WORD` opcode, to account for the difference
+in lengths of the preimages.
+The `hashcost` is deducted at the same time as memory-expansion gas and `CreateGas` is deducted: _before_ evaluation
+of the resulting address and the execution of `init_code`.
+
+- `0xfe` is a single byte, 
+- `address` is always `20` bytes, 
+- `salt` is always `32` bytes (a stack item). 
+
+The preimage for the final hashing round is thus always exactly `53` bytes long (does not include hash of the init code).
+
+### Motivation
+
+The core motivations are the same as [./eip-1014.md](EIP-1014), i.e. deploying a smart contract at a deterministic address.
+This variation removes the hash of the init code from the preimage. The issue with the existing `CREATE2` opcode
+is that computing the hash of the init code on-chain requires the entire bytecode of the target contract to be included
+in the contract computing the first preimage.
+
+### Rationale
```

Comment:
Top level sections should have `##` and subsections should have `###`.  This is a problem throughout this document.  See https://raw.githubusercontent.com/ethereum/EIPs/master/eip-template.md for an example layout.